pad.c

```diff
@@ -1975,7 +1976,8 @@ S_cv_clone_pad(pTHX_ CV *proto, CV *cv, CV *outside, HV *cloned,
     PL_compcv = cv;
     if (newcv) SAVEFREESV(cv); /* in case of fatal warnings */
 
-    CvOUTSIDE(cv) = CvREFCNT_inc_simple(outside);
+    if (CvHASEVAL(cv))
```

Comment:
looks like CvREFCNT_inc_simple isn't a thing in v5.40.0

Reply:
> looks like CvREFCNT_inc_simple isn't a thing in v5.40.0

I added it quite recently. For a backport you can just use regular `SvREFCNT_inc` and cast its return value

```c
CvOUTSIDE(cv) = (CV *)SvREFCNT_inc(outside);
```

Reply:
yep it's what I'm doing in my patch. It would be really good to see this backport and there to be a 5.40.1 given we're reverting a behavior that only breaks in 5.40 right now.